src/components/performance/register-modal.tsx

```
@@ -0,0 +1,173 @@
+        className={`
+          h-175 w-300 max-w-none rounded-lg border-none p-[80px_122px] shadow-xl
+        `}
+        showCloseButton={false}
+      >
+        <button
+          onClick={onClose}
+          className={`
+            absolute top-8 right-8 text-gray-400
+            hover:text-black
+          `}
+        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)